src/plugins/sql-support/plugin.js

```diff
@@ -389,9 +389,14 @@ QueryBuilder.extend({
                 var opVal = sqlrl.call(this, value, data.operation);
                 if (operator == 'NOT LIKE') opVal.op = 'not_' + opVal.op;
 
+                var left_value = data.left.value.value;
```

Comment:
So I tested and `data.left.value.value` NEVER exists. But `data.left.values` ALWAYS exists, even with fields without table prefix. That's why your code works. So you can just keep line 394 and remove everything else.